magit-arcanist-diff.el

```diff
@@ -36,7 +35,8 @@
   "Popup console for Arcanist diff commands."
   :switches '((?l "No lint" "--nolint")
               (?u "No unit tests" "--nounit")
-              (?c "No coverage info" "--no-coverage"))
+              (?c "No coverage info" "--no-coverage")
+              (?b "browse" "--browse"))
```

Comment:
There is one potential issue here, given the linter triggered by `arc diff` produces some changes to be made there is no way to interact and approve or dismiss those changes the process will just hang. The way to go here is to trigger user interaction in this case, need to figure out how to do this for a process being run via magit.